src/tests/fs_tests.rs

```diff
@@ -948,4 +949,31 @@ pub mod fs_tests {
         assert_eq!(execcage.exit_syscall(EXIT_SUCCESS), EXIT_SUCCESS);
         lindrustfinalize();
     }
+
+
+    pub fn ut_lind_fs_shm() {
+        lindrustinit(0);
+        let cage = {CAGE_TABLE.get(&1).unwrap().clone()};
+        let key = 31337;
+        let mut shmidstruct = ShmidsStruct::default();
+
+        // shmget returns an identifier in shmid
+        let shmid = cage.shmget_syscall(key, 1024, 0666|IPC_CREAT);
+    
+        // shmat to attach to shared memory
+        cage.shmat_syscall(shmid, 0xffffff60 as *mut u8, 0);
```

Comment:
we don't want this and other function's return values?

Reply:
You're right is could be done better, also pointed out an error to me, addresses don't match in shmat/shmdt